tools/codegen/base/xbuild.json

```diff
@@ -0,0 +1,12 @@
+{
+  "repository": "https://github.com/rust-mobile/xbuild",
+  "tag_prefix": "v",
+  "rust_crate": "${package}",
+  "asset_name": "${package}-${os_name}-x64${exe}",
+  "bin": "${package}-${os_name}-x64${exe}",
+  "platform": {
+    "x86_64_linux_musl": {},
```

Comment:
IIUC, this is actually not linux-musl, is linux-gnu.
https://github.com/rust-mobile/xbuild/blob/c2f3291a02e8dddf531b2583f8e15b82f1461c61/.github/workflows/release.yml#L13